README.md

```diff
@@ -241,9 +241,9 @@ following rules are enabled by default:
 * `git_not_command` &ndash; fixes wrong git commands like `git brnch`;
 * `git_pull` &ndash; sets upstream before executing previous `git pull`;
 * `git_pull_clone` &ndash; clones instead of pulling when the repo does not exist;
-* `git_pull_uncommitted_changes` &ndash; stashes changes before pulling and pops them afterwards;
+* `git_pull_uncommitted_changes` &ndash; stashes changes before pulling and pops them afterward;
```

Comment:
"afterwards" is a correct usage, if I'm not mistaken.
```suggestion
* `git_pull_uncommitted_changes` &ndash; stashes changes before pulling and pops them afterwards;
```

Reply:
Are you Canadian?

Reply:
Maybe I am (and probably the the athor of the original text is too) more used to British English 😊

Reply:
Seems legit, I'm just used to American